Client/src/Components/Inputs/Checkbox/index.jsx

```diff
@@ -66,11 +67,11 @@ const Checkbox = ({
 			sx={{
```

Comment:
To solve the padding rpoblem, delete the padding bottom and add alignSelf: "flex-start" to the sx of the MuiCheckbox (before &:hover, on line 59 or 60